Geometry/HGCalCommonData/interface/HGCalGeomTools.h

```diff
@@ -10,15 +10,13 @@ class HGCalGeomTools {
   HGCalGeomTools();
   ~HGCalGeomTools() {}
 
-  enum WaferType {
-    WaferFull = 0,
-    WaferFive = 1,
-    WaferChoptwo = 2,
-    WaferChopTwom = 3,
-    WaferHalf = 4,
-    WaferSemi = 5,
-    WaferSemi2 = 6,
-    WaferThree = 7
+  enum WaferCorner {
```

Comment:
`enum class` is preferred, if it will work for this case.

Reply:
It does not work for this case